R/importNGWMN_wml2.R

```diff
@@ -25,8 +25,9 @@
 #' "observedProperty=urn:ogc:def:property:OGC:GroundWaterLevel",
 #' "responseFormat=text/xml",
 #' "featureOfInterest=VW_GWDP_GEOSERVER.USGS.403836085374401",sep="&")
-#' data <- importNGWMN(obs_url)
-#' 
+#' if(!httr::http_error(obs_url)){
+#'   data_returned <- importNGWMN(obs_url)
+#' }
```

Comment:
Does this hit that URL twice?

Reply:
This was the original issue because the cida link was 502'ing. I added this check because it's the sketchiest source. I could take that out though...